src/main/java/org/sonarsource/sonarlint/ls/CommandManager.java

```diff
@@ -122,7 +122,8 @@ public class CommandManager {
     SONARLINT_OPEN_STANDALONE_RULE_DESCRIPTION_COMMAND,
     SONARLINT_BROWSE_TAINT_VULNERABILITY,
     SONARLINT_SHOW_TAINT_VULNERABILITY_FLOWS,
-    SONARLINT_SUGGEST_FIX_COMMAND);
+    SONARLINT_SUGGEST_FIX_COMMAND,
+    SONARLINT_SUGGEST_TAINT_FIX_COMMAND);
```

Comment:
This change is not related to SLLS-354. It is a separate change needed for AI CodeFix to work with taints.